packages/components/tooltip/src/TooltipContent.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Box, type CommonProps, type ExpandProps } from '@contentful/f36-core';
+import {
+  FloatingArrow,
+  FloatingPortal,
+  useMergeRefs,
+} from '@floating-ui/react';
+import { useTooltipContext } from './TooltipContext';
+import { WithEnhancedContent } from './Tooltip';
+import { getTooltipContentStyles } from './Tooltip.styles';
+import { cx } from '@emotion/css';
+
+type TooltipContentProps = CommonProps &
+  WithEnhancedContent & { id?: string; as?: React.ElementType };
+
+export const TooltipContent = React.forwardRef<
+  HTMLDivElement,
+  ExpandProps<TooltipContentProps>
+>(({ content, label, style, className, id, ...otherProps }, propRef) => {
+  const state = useTooltipContext();
+  const ref = useMergeRefs([state.refs.setFloating, propRef]);
+  const styles = getTooltipContentStyles(state.isOpen);
+  if (!state.isOpen) {
+    return null;
+  }
+
+  const wrappedTooltipContents = content ? (
+    <Box
+      {...state.getFloatingProps()}
+      {...otherProps}
+      className={cx(styles.tooltip, className)}
+      style={{ ...style, ...state.floatingStyles }}
+      ref={ref}
+      id={id}
+    >
+      <span aria-label={label}>{content}</span>
```

Comment:
Aria labels can't be used on static elements

```suggestion
      <span>{content}</span>
```